### PR TITLE
Remove Connection documentation; where not already duplicated, move d…

### PR DESCRIPTION
**Pull request by andy-faunadb**
where not already duplicated, move docs to Client

### CODE REVIEW

File: lib/fauna/client.rb [hunk 1]
```diff
@@ -59,10 +59,10 @@ def post(path, data = {})
     end
 
     ##
-    # Performs a PUT request for a REST endpoint.
+    # Performs a +PUT+ request for a REST endpoint.
```
Comment (ashfire908) [inline lib/fauna/client.rb]: Why not put these on `PATCH` and `DELETE`?

File: lib/fauna/connection.rb [hunk 2]
```diff
@@ -1,105 +1,48 @@
...
-      @port = params[:port] || (@scheme == 'https' ? 443 : 80)
-      @timeout = params[:timeout] || 60
-      @connection_timeout = params[:connection_timeout] || 60
-      @adapter = params[:adapter] || Faraday.default_adapter
-      @credentials = params[:secret].to_s.split(':', 2)
+      domain = params[:domain] || 'rest.faunadb.com'
+      scheme = params[:scheme] || 'https'
+      port = params[:port] || (scheme == 'https' ? 443 : 80)
+      timeout = params[:timeout] || 60
+      # TODO: this isn't documented...
```
Comment (ashfire908) [inline lib/fauna/connection.rb]: There's docs for the param, what's missing?

File: lib/fauna/client.rb (hunk 1 above)
Comment (andy-faunadb) [inline lib/fauna/client.rb]: Done

**Comment by ashfire908** [comment]
LGTM

**Pull request merged**